build.xml

```diff
@@ -75,7 +75,7 @@
         <!-- java compiler properties -->
         <property name="javac.source" value="1.6" />
         <property name="javac.target" value="1.6" />
-        <property name="bootclasspath.dir" value="C:\Program Files (x86)\Java\jdk1.6.0_45\jre\lib" />
+       <property name="bootclasspath.dir" value="D:\Documents\EclipseProjects\RomRaider"/>
```

Comment:
This needs to resolve to the location of the Java 1.6 rt.jar, please remove.

Reply:
When I use the default build.xml I cannot compile it. I could compile it with other versions than 1.6 before, is 1.6 necessary for compilation now?

Reply:
Ah okay, so after some search this is needed so not only 1.6 rules are followed but also no new libraries are included

Reply:
Yes, without it and using Java 8 to cross compiling to 6, a few Java 8 references will creep in.  This is the default location for Java 6 JDK.  Adjust this locally for your own builds but I'd like to keep this default in the repo.